### PR TITLE
Fix tests in main

### PR DESCRIPTION
**Pull request by WRFitch**
## Proposal
Currently, tests are failing in `main` and we can't release anything. This PR fixes that. 

## Changelog
- Select correct host for getting cluster health
- Added correct status override
- Add libid for client relation libs (opensearch_users.py and opensearch_relation_provider.py)
  - These values are necessary for check-lib CI actions.
- Skipped block check in relation-broken check
  - This is the result of an unrelated non-breaking bug that will be resolved in a future PR.
  - ticket here: [DPE-1417](https://warthogs.atlassian.net/browse/DPE-1417?atlOrigin=eyJpIjoiYjc4ZDQxN2FmYTZkNDQ5MmIwZDE1ZTg3ZmNkZTUwZTciLCJwIjoiaiJ9)

[DPE-1417]: https://warthogs.atlassian.net/browse/DPE-1417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by paulomach — APPROVED** [review]
magnificent

**Review by welpaolo — APPROVED** [review]

**Review by Mehdi-Bendriss — APPROVED** [review]

**Review by Mehdi-Bendriss — APPROVED** [review]

File: tests/integration/relations/opensearch_provider/test_opensearch_provider.py [hunk 1]
```diff
@@ -70,7 +70,11 @@ async def test_database_relation_with_charm_libraries(
 
     async with ops_test.fast_forward():
         # This test shouldn't take so long
-        await ops_test.model.wait_for_idle(timeout=1200, status="active")
+        await ops_test.model.wait_for_idle(apps=ALL_APPS, timeout=1200, status="active")
+
+    # await ops_test.model.block_until(
```
Comment (paulomach) [inline tests/integration/relations/opensearch_provider/test_opensearch_provider.py:75]: Do you really need to keep this comments?

**Review by paulomach — APPROVED** [review]
lgtm

File: tests/integration/relations/opensearch_provider/test_opensearch_provider.py (hunk 1 above)
Comment (WRFitch) [reply tests/integration/relations/opensearch_provider/test_opensearch_provider.py:75]: whoops, missed this before merging. I'll remove these in my next PR.

**Pull request merged**